lib/install/management/store_files.go

```diff
@@ -72,19 +71,47 @@ func (d *Dispatcher) DeleteStores(vchVM *vm.VirtualMachine, conf *config.Virtual
 	return nil
 }
 
-func (d *Dispatcher) deleteParent(ds *object.Datastore, root string) error {
-	defer trace.End(trace.Begin(""))
+func (d *Dispatcher) deleteImages(conf *config.VirtualContainerHostConfigSpec) error {
+	var emptyImages bool
+	var errs []string
+
+	for _, imageDir := range conf.ImageStores {
+		imageDSes, err := d.session.Finder.DatastoreList(d.ctx, imageDir.Host)
+		if err != nil {
+			errs = append(errs, err.Error())
+			continue
+		}
+
+		if len(imageDSes) != 1 {
+			errs = append(errs, fmt.Sprintf("Found %d datastores with provided datastore path %s. Provided datastore path must identify exactly one datastore.",
```

Comment:
omg lol @fdawg4l I just actually laughed IRL at that response.

 Back to the topic though, is this error message a bit better?